wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -487,6 +487,29 @@ public void createSnapshot(String workspaceId) throws NotFoundException, ServerE
         return machineManager.getSnapshots(sessionUser().getId(), workspaceId);
     }
 
+    /**
+     * Adds machine into workspace runtime.
```

Comment:
I think that you should add more useful information in the javadoc. e.g. What if workspace is stopped, will this machine be stopped with the workspace. Will this method affect the workspace configuration?